AcceptanceTests/AcceptanceTests.swift

```diff
@@ -235,6 +256,16 @@ extension AcceptanceTests {
         return contentsOfFileAsString(at: muterHelpOutputPath)
     }
     
+    var muterInitHelpOutputPath: String { return "\(AcceptanceTests().rootTestDirectory)/muters_init_help_output.txt" }
```

Comment:
Out of scope, unless you want to fix it now while you're in here: these could all be `let`s, or at the very least, get rid of `return` and put each one on one line

Reply:
oh, these are in an extension :(